crates/generator/src/generator.rs

```diff
@@ -734,6 +742,7 @@ impl Generator {
         let backend = self
             .load_backend(block_info.number().unpack(), state, &script_hash)
```

Comment:
Why not load `max_cycles` here?

Reply:
OK. I see that debug max cycles is from RPC params.

So how about taking `override_max_cycles: Option<u64>`, and load from config if it's `None`.

Reply:
Good idea